packages/kbn-axe-config/index.ts

```diff
@@ -54,5 +54,8 @@ export const AXE_OPTIONS = {
     bypass: {
       enabled: false, // disabled because it's too flaky
     },
+    'nested-interactive': {
+      enabled: false, // tracker here - https://github.com/elastic/kibana/issues/152494 disabled because we have too many failures on interactive controls
+    },
```

Comment:
Bhavya and I discussed ignoring this rule. It's not ideal but the right call in the name of getting access to the updated rule set.